### PR TITLE
Newtonsoft and Layout Changes

### PR DESCRIPTION
**Pull request by marmegh**
# Updates
- Added images for Sonar and For All Secure to the contributor list
image
- Replaced WhiteSource contributor image with `Mend.png` reflecting name change
- Layout updates and added java-sarif link
- Newtonsoft.Json update to address [Advisory: Improper Handling of Exceptional Conditions in Newtonsoft.Json](https://github.com/advisories/GHSA-5crp-9r3c-p9vr). This includes updating Sarif.Multitool.Library to v3.0.0-beta1 (which directly implements the mitigation without updating to Newtonsoft 13.0) and Microsoft.ApplicationInsights.AspNetCore to 2.21.0.

# [Sarif.sdk](https://github.com/microsoft/sarif-sdk)
<details>

# SARIF Package Release History (SDK, Driver, Converters, and Multitool)

## 3.0.0-beta1 [Sdk](https://www.nuget.org/packages/Sarif.Sdk/3.0.0-beta1) | [Driver](https://www.nuget.org/packages/Sarif.Driver/3.0.0-beta1) | [Converters](https://www.nuget.org/packages/Sarif.Converters/3.0.0-beta1) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/3.0.0-beta1) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/3.0.0-beta1)
* BUGFIX: Loosen Newtonsoft.JSON minimum version requirement to 6.0.8 (for .NET framework) or 9.0.1 (for all other compilations) for Sarif.Sdk. Sarif.Converts requires 8.0.1, minimally, for .NET framework compilations.
* BUGFIX: Broaden set of supported .NET frameworks for compatibility reasons. Sarif.Sdk now supports net45 forward. Sarif.Driver and Sarif.WorkItems requires net461 due to other dependencies.
* BUGFIX: Set default stack limit in Newtonsoft.JSON utilization (if `JsonConvert.Defaults` is not already configured) to address GitHub advisory [GHSA-5crp-9r3c-p9vr](https://github.com/advisories/GHSA-5crp-9r3c-p9vr).

## **v2.4.16** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.4.16) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.4.16) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.4.16) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.4.16) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/2.4.16)

* FEATURE: Add `max-file-size-in-kb` argument that allows filtering scan targets by file size. [#2494](https://github.com/microsoft/sarif-sdk/pull/2494)
* BUGFIX: Fix false positive for `SARIF1002.UrisMustBeValid` for file URIs that omit the `authority`. [#2501](https://github.com/microsoft/sarif-sdk/pull/2501)
* DEPENDENCY BREAKING: SARIF now requires Newtonsoft.JSON 13.0.1. Updating [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/13.0.1) to v13.0.1, [Microsoft.Json.Schema](https://www.nuget.org/packages/Microsoft.Json.Schema) to v1.1.5, [Microsoft.Json.Pointer](https://www.nuget.org/packages/Microsoft.Json.Pointer) to v1.1.5, [Microsoft.Azure.Kusto.Data](https://www.nuget.org/packages/Microsoft.Azure.Kusto.Data) to v10.0.3, [Microsoft.NET.Test.Sdk](https://www.nuget.org/packages/Microsoft.NET.Test.Sdk/17.4.0-preview-20220707-01) to v17.4.0-preview-20220707-01, [Microsoft.Extensions.Logging.ApplicationInsights](https://www.nuget.org/packages/Microsoft.Extensions.Logging.ApplicationInsights/2.20.0) to v.2.20.0, [Microsoft.TeamFoundationServer.Client](https://www.nuget.org/packages/Microsoft.TeamFoundationServer.Client/16.170.0) to v.16.170.0, [Microsoft.Coyote](https://www.nuget.org/packages/Microsoft.Coyote) to v.1.5.8 and [Microsoft.Coyote.Test](https://www.nuget.org/packages/Microsoft.Coyote.Test) to v.1.5.8 in response to [Advisory: Improper Handling of Exceptional Conditions in Newtonsoft.Json](https://github.com/advisories/GHSA-5crp-9r3c-p9vr). [#2504](https://github.com/microsoft/sarif-sdk/pull/2504)
</details>
<br />

</details>

### CODE REVIEW

File: src/SarifWeb/SarifWeb.csproj [hunk 1]
```diff
@@ -42,8 +42,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.20.0" />
-    <PackageReference Include="Sarif.Multitool.Library" Version="2.4.14" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
+    <PackageReference Include="Sarif.Multitool.Library" Version="2.4.16" />
```
Comment (marmegh) [inline src/SarifWeb/SarifWeb.csproj:46]: @EasyRhinoMSFT, do we need something other than this to update the Multitool package?

File: src/SarifWeb/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -56,8 +56,14 @@
         The SARIF project is supported by a group of industry contributors.
     </div>
     <div id="collabSectionButtons" role="list" aria-labelledby="collabSectionTitle">
-        <a role="listitem" href="http://www.kestreltechnology.com" target="_blank">
-            <img aria-label="Kestrel Technology" src="~/Images/Logos/KestrelTechnology.png" alt="Kestrel Technology" />
+        <a href="https://www.sonarsource.com/" target="_blank">
+            <img role="listitem" aria-label="Sonar" src="~/Images/Logos/Sonar.png" alt="Sonar" />
+        </a>
+        <a href="https://forallsecure.com/" target="_blank">
+            <img role="listitem" aria-label="For All Secure" src="~/Images/Logos/ForAllSecure.png" alt="For All Secure" />
```
Comment (EasyRhinoMSFT) [inline src/SarifWeb/Views/Home/Index.cshtml:63]: >For All Secure [](http://example.com/codeflow?start=45&length=14)

This is one word, ForAllSecure

File: src/SarifWeb/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -159,7 +162,9 @@
                         </svg>
                     </div>
                 </div>
-                <div class="tools-item-description tools-other-item-description"><a href="https://github.com/microsoft/sarif-js-sdk">Code and supporting files</a> for working with SARIF in JavaScript applications.</div>
+                <div class="tools-item-description tools-other-item-description">
+                    <a href="https://github.com/microsoft/sarif-js-sdk">Code and supporting files</a> for working with SARIF in JavaScript applications.
+                    </div>
```
Comment (EasyRhinoMSFT) [inline src/SarifWeb/Views/Home/Index.cshtml:167]: >     [](http://example.com/codeflow?start=16&length=4)

Remove

File: src/SarifWeb/SarifWeb.csproj (hunk 1 above)
Comment (EasyRhinoMSFT) [reply src/SarifWeb/SarifWeb.csproj:46]: Nope this should be it.
Note: using the Package Manager is the best way to go and ensures that everything is updated correctly.

Comment (marmegh) [reply src/SarifWeb/SarifWeb.csproj:46]: Unfortunately, this approach doesn't work in most of the repos for this particular update. :(

Comment (EasyRhinoMSFT) [reply src/SarifWeb/SarifWeb.csproj:46]: What is the issue?

**Review by EasyRhinoMSFT — APPROVED** [review]
:shipit:

File: src/SarifWeb/SarifWeb.csproj (hunk 1 above)
Comment (marmegh) [reply src/SarifWeb/SarifWeb.csproj:46]: No issue, this thread started in a Teams chat. I think some of the context got lost.

**Review by EasyRhinoMSFT — APPROVED** [review]

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -2,4 +2,4 @@
 # This file controls who is tagged for review for any given pull request.
 
 # For anything not explicitly taken by someone else:
-* @eddynaka @michaelcfanning @EasyRhinoMSFT
\ No newline at end of file
+* @michaelcfanning @EasyRhinoMSFT @marmegh
```
Comment (michaelcfanning) [inline .github/CODEOWNERS:5]: >michaelcfanning [](http://example.com/codeflow?start=3&length=15)

might want to add cfaucon as a defensive measure. :)

File: src/ReleaseHistory.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # SARIF Website Release History
 
+## **v1.0.1**
+- Updating contributors list/images, bump Sarif.Multitool.Library from 2.4.14 to 2.4.16, add link to java-sarif documentation. [#142](https://github.com/microsoft/sarif-website/pull/142)
```
Comment (michaelcfanning) [inline src/ReleaseHistory.md:4]: >contributors [](http://example.com/codeflow?start=11&length=12)

Just curious, didn't we hear from Contrast Security that they'd be willing to be badged on the site?

**Review by michaelcfanning — APPROVED** [review]
:shipit:

File: src/ReleaseHistory.md (hunk 5 above)
Comment (marmegh) [reply src/ReleaseHistory.md:4]: Possibly, but to my knowledge they have not submitted the graphic for inclusion at this time. @EasyRhinoMSFT, have you heard from them at all?

**Pull request merged**